change/@fluentui-react-charting-14c34b26-6050-4fa3-8ab9-ccfc1661f953.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33519&commitId=5765a28b660af7daf978240547623e3ea94cced6&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|
|Callout.Top auto edge.chromium.png|2307| Changed|
|Callout.Bottom center.chromium.png|2146| Changed|
|Callout.Bottom Right Edge RTL.chromium.png|1593| Changed|
|Callout.Right bottom edge.chromium.png|3005| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
VR unrelated to changes in this PR.